lm_eval/models/optimum_lm.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+from lm_eval.api.registry import register_model
+from lm_eval.models.huggingface import HFLM
+
+try:
+    import optimum    
+except ModuleNotFoundError: 
+    raise Exception("package `optimum` is not installed. Install it via `pip install optimum[openvino] ipywidgets pillow torchaudio`")
+from optimum.intel.openvino import OVModelForCausalLM
+
+@register_model("optimum-causal")
+class OptimumLM(HFLM):
```

Comment:
Please add a test that I had in the first integration: https://github.com/EleutherAI/lm-evaluation-harness/pull/994/files